### ISSUE TITLE
Add more HalfPyramids

### ISSUE THREAD

**Issue by yashlamba** [ricksanc/cpp-programs#5]
Below are the half pyramids to be added:

1. Inverted Half Pyramid of *s : For height entered by user, print half pyramid like in the example
``` 
For Height = 5

O/P:
*****
****
***
**
*
```
2. Inverted Half Pyramid using numbers : For height entered by user, print half pyramid like in the example
``` 
For Height = 5

O/P:
1 2 3 4 5
1 2 3 4
1 2 3
1 2
1
```
3. Floyd's Triangle Half Pyramid : For height entered by user, print half pyramid like in the example
``` 
For Height = 5

O/P:
1
2 3
4 5 6
7 8 9 10 
11 12 13 14 15
```

**Comment by yashlamba** [comment]
Floyd's Pyramid not printing from 1

**Comment by sdhanshu** [comment]
Fixed that problem in commit b4af0ba

**Comment by yashlamba** [comment]
Great Work!
